src/app/home-page/home-logged-out/home.component.html

```diff
@@ -54,10 +54,14 @@ <h2>An app store for bioinformatics</h2>
             aria-label="Search"
             #search
             type="text"
+            [(ngModel)]="search.value"
```

Comment:
not entirely sure this is needed.

Reply:
If I delete this line the x icon only shows up the first time I enter something in the input

Reply:
odd...but anyways, assuming you always want the x to show up (i prefer not), try removing this and the *ngIf on the button then

Reply:
okay, thank you I'll try that out. The x doesn't always show, just only when you enter something into the search